_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,142 @@
+---
+id: 0va7u6
+name: HTML graphics contain no text
+rule_type: atomic
+description: |
+  This rule checks that images of text are not used
+accessibility_requirements:
+  wcag20:1.4.5: # Images of Text (AA)
```

Comment:
It should probably also map to 1.4.9 (+ possibly need for a note in background saying that examples are designed for 1.4.5 and Failed Examples may not fail 1.4.9, see similar note in color contrast rule).

(+ we might have another "free" rule for 1.4.9 once this is done…)

Reply:
Included 1.4.9. Good suggestion